features/HTTP.feature

```diff
@@ -236,19 +236,21 @@ Feature: Test HTTP API
 
   @usingHttp @cleanSecurity
   Scenario: Create/get/search/update/delete role
-    When I create a new role "role1" with id "test"
+    When I create a new role "foo" with id "test"
     Then I'm able to find a role with id "test"
-    And I update the role with id "test" with role "role2"
-    Then I'm able to find a role with id "test" with role "role2"
-    Then I'm able to find "1" role by searching controller corresponding to role "role2"
+    And I update the role with id "test" with role "bar"
```

Comment:
Yes, it's exaclty that.
This sentence was still there, I just changed the role's name.